### PR TITLE
feat: add dry run tests for arbitrum and primary

### PR DESCRIPTION
**Pull request by cehali**

### CODE REVIEW

File: .github/workflows/oracle-avalanche-dry-run-test.yml [hunk 1]
```diff
@@ -24,7 +24,6 @@ jobs:
           COINGECKO_API_URL: https://pro-api.coingecko.com/api/v3/simple/price
           COINGECKO_API_KEY: ${{ secrets.COINGECKO_API_KEY }}
           COINMARKETCAP_API_KEY: ${{ secrets.COINMARKETCAP_API_KEY }}
-          KAIKO_API_KEY: ${{ secrets.KAIKO_API_KEY }}
```
Comment (rpiszczatowski) [inline .github/workflows/oracle-avalanche-dry-run-test.yml:27]: Why this change?

**Review by rpiszczatowski — APPROVED** [review]

File: .github/workflows/oracle-avalanche-dry-run-test.yml (hunk 1 above)
Comment (cehali) [reply .github/workflows/oracle-avalanche-dry-run-test.yml:27]: We don't use kaiko in avalanche manifest, this API key is not required here
